prompt/prompt-model/src/commonMain/kotlin/ai/koog/prompt/message/Message.kt

```diff
@@ -83,7 +94,8 @@ public sealed interface Message {
     @Serializable
     public data class Assistant(
         override val content: String,
-        val finishReason: String? = null
+        val finishReason: String? = null,
```

Comment:
I think you can move `finishReason` to metadata class now, it looks like it fits there

Reply:
But you already have separate metadata for request and response. Why don't you want to extend it further and make a special metadata for assistant messages too, doesn't seem to bad to me